.changeset/nervous-steaks-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/treeview": patch
+---
+
+Use custom property in place of direct value when defining mod.
```

Comment:
```suggestion
Use the already defined custom property `--spectrum-treeview-item-border-color-quiet-selected` in place of the direct value. The border color value remains the same.
```
This goes into the change log, and it'd be unclear what properties and values and mods this is talking about.

Reply:
@cdransf Did this get updated already?